v2/split_slice.go

```diff
@@ -0,0 +1,29 @@
+package pie
+
+// SplitSlice returns splited slices which length equals splitLength.
```

Comment:
"... returns slices with each length equalling splitLength, except for the last slice which may contain less. The order is maintained in each of the slices and slices themselves."

Reply:
ok